solutions/experimentation/core-landing-zone/lz-folder/audits/logging-project/project-sink.yaml

```diff
@@ -24,7 +24,7 @@ metadata:
   name: logging-project-id-data-access-sink # kpt-set: ${logging-project-id}-data-access-sink
   namespace: logging
   annotations:
-    config.kubernetes.io/depends-on: security-log-bucket # kpt-set: logging.cnrm.cloud.google.com/namespaces/logging/LoggingLogBucket/${security-log-bucket}
+    config.kubernetes.io/depends-on: security-log-bucket # kpt-set: logging.cnrm.cloud.google.com/namespaces/logging/LoggingLogBucket/security-log-bucket
```

Comment:
I think the kpt-set can be removed and the value can become : logging.cnrm.cloud.google.com/namespaces/logging/LoggingLogBucket/security-log-bucket

Reply:
That's a good catch; I missed it when trying to mirror the changes I did in the core landing zone. Thanks. Please resolve if you're happy.